pkg/shell.go

```diff
@@ -36,7 +40,7 @@ func ExecShellWithVars(osvars map[string]string, command string, args ...string)
 	for k, v := range osvars {
 		os.Setenv(k, v)
 		suppressedValue := strings.Repeat("*", len(v))
-		log.Info().Msgf(" export %s = %s", k, suppressedValue)
+		log.Debug().Msgf(" export %s = %s", k, suppressedValue)
 	}
```

Comment:
We need this at info level.

This allow us to know if we forgot to pass something to terraform

Reply:
my understanding is that if you need that for development, it's a debug data, end user won't care about it (info level).  but I'll follow your requests to unblock the PR